packages/contentstack-import/src/import/modules-js/global-fields.js

```diff
@@ -88,7 +88,7 @@ module.exports = class ImportGlobalFields {
                   // eslint-disable-next-line no-undef
                   log(self.config, `Globalfield '${snip.uid} already exists'`, 'error');
                 } else {
-                  log(self.config, chalk.red(`Globalfield failed to import ${formatError(error)}`), 'error');
+                  log(self.config, chalk.red(`Globalfield ${snip.title}failed to import ${formatError(error)}`), 'error');
```

Comment:
@cs-raj Add a whitespace after the title} in `${snip.title}failed to import`

Reply:
Yes @netrajpatel